tools/cli/src/mentat_cli/repl.rs

```diff
@@ -52,21 +52,22 @@ impl Repl {
 
     /// Runs the REPL interactively.
     pub fn run(&mut self) {
-        let mut more: Option<Command> = None;
+        let mut more = false;
```

Comment:
There is a compiler warning with this patch sequence that reveals that `more` is only assigned to, not read.  This is vestigial?